records_mover/records/pandas/format_for_csv.py

```diff
@@ -0,0 +1,62 @@
+from pandas import DataFrame
+import pandas as pd
+from records_mover.records.schema import RecordsSchema
+from records_mover.records import DelimitedRecordsFormat
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def format_df_for_csv_output(df: DataFrame,
+                             records_schema: RecordsSchema,
+                             records_format: DelimitedRecordsFormat) -> DataFrame:
+    formatted_df = df.copy(deep=False)
+
+    logger.info(f"VMB formatted_df: {formatted_df}")
+    logger.info(f"VMB index, field: {list(enumerate(records_schema.fields))}")
+    logger.info(f"VMB formatted_df.iloc[:, 1]: {formatted_df.iloc[:, 1]}")
+    for index, field in enumerate(records_schema.fields):
+        logger.info(f"VMB: Looking at field {field.name} with type {field.field_type}")
+        if field.field_type == 'date':
+            # TODO: is there a less weird way of doing this?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker